wiki/Guides/Getting_Your_Map_Modded/th.md

```diff
@@ -0,0 +1,39 @@
+---
+outdated: true
+---
+
+<!-- TODO: modreqs doesn't work anymore -->
+
+# วิธีการทำให้แมพเพลงของคุณถูกตรวจสอบข้อบกพร่อง
```

Comment:
```suggestion
# วิธีการทำให้บีทแมพของผ่านการวิจารณ์บีทแมพ
```
The "modding" is translated as "การวิจารณ์บีทแมพ" according to previous articles and the osu!web
So if possible use "การวิจารณ์บีทแมพ" for consistency